sla-template-descriptor/examples/sla-template-example.yml

```diff
@@ -0,0 +1,83 @@
+---
+descriptor_schema: "https://raw.githubusercontent.com/sonata-nfv/tng-schema/master/service-descriptor/nsd-schema.yml"
```

Comment:
This should point to the sla schema. So "https://raw.githubusercontent.com/sonata-nfv/tng-schema/master/sla-template-descriptor/sla-template-schema.yml"